content/akeneo-web-api.yaml

```diff
@@ -1267,6 +1267,140 @@ paths:
         422:
           $ref: "#/responses/422Error"
 
+  /attribute-groups:
+    get:
+      summary: Get list of attribute groups
+      operationId: "attribute_groups_get_list"
+      description: This endpoint allows you to get a list of attribute groups. Attribute groups are paginated and sorted by code.
+      tags:
+        - Attribute groups
+      parameters:
+        - $ref: '#/parameters/page'
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/with_count'
+      responses:
+        200:
+          description: Return attribute groups paginated
+          schema:
+            title: Attribute groups
+            type: object
+            allOf:
+              - $ref: '#/definitions/Pagination'
+              - properties:
+                  _embedded:
+                    type: object
+                    properties:
+                      _items:
+                        type: array
+                        items:
+                          $ref: '#/definitions/AttributeGroupList'
+          examples: {
+            _links: {
+              self: {
+                href: "https://demo.akeneo.com/api/rest/v1/attribute-groups?page=3&limit=2"
+              },
+              first: {
+                href: "https://demo.akeneo.com/api/rest/v1/attribute-groups?page=1&limit=2"
+              },
+              previous: {
+                href: "https://demo.akeneo.com/api/rest/v1/attribute-groups?page=2&limit=2"
+              },
+              next: {
+                href: "https://demo.akeneo.com/api/rest/v1/attribute-groups?page=4&limit=2"
+              }
+            },
+            current_page: 3,
+            _embedded: {
+              _items: [
+                {
+                  "_links":{
+                     "self":{
+                        "href":"http://localhost/api/rest/v1/attribute-groups/attributeGroupA"
+                     }
+                  },
+                  "code":"attributeGroupA",
+                  "sort_order":1,
+                  "attributes":[
+                     "sku",
+                     "a_date",
+                     "a_file",
+                     "a_price",
+                     "a_price_without_decimal",
+                     "a_ref_data_multi_select",
+                     "a_ref_data_simple_select",
+                     "a_text",
+                     "a_regexp",
+                     "a_text_area",
+                     "a_yes_no",
+                     "a_scopable_price",
+                     "a_localized_and_scopable_text_area"
+                  ],
+                  "labels":{
+                     "en_US":"Attribute group A",
+                     "fr_FR":"Groupe d\u0027attribut A"
+                  }
+                },
+                {
+                  "_links":{
+                     "self":{
+                        "href":"http://localhost/api/rest/v1/attribute-groups/attributeGroupB"
```

Comment:
Root URL are not consistant throughout your example.
We used "https://demo.akeneo.com" for now.